File: app/src/main/java/org/neidhardt/dynamicsoundboard/misc/Logger.java
package org.neidhardt.dynamicsoundboard.misc;

import android.util.Log;
import org.neidhardt.dynamicsoundboard.SoundboardConfiguration;

/**
 * Project created by eric.neidhardt on 27.08.2014.
 */
public class Logger
{
	public static void e(String TAG, String msg)
	{
		if (SoundboardConfiguration.ENABLE_LOGGING)
			Log.e(TAG, msg);
	}

	public static void d(String TAG, String msg)
	{
		if (SoundboardConfiguration.ENABLE_LOGGING)
			Log.d(TAG, msg);
	}

}

File: app/src/main/kotlin/org/neidhardt/dynamicsoundboard/misc/Logger.kt
package org.neidhardt.dynamicsoundboard.misc

import android.util.Log
import org.neidhardt.dynamicsoundboard.SoundboardConfiguration

/**
 * Project created by eric.neidhardt on 27.08.2014.
 */
object Logger {

	fun e(TAG: String, msg: String?) {
		if (SoundboardConfiguration.ENABLE_LOGGING)
			Log.e(TAG, msg)
	}

	fun d(TAG: String, msg: String?) {
		if (SoundboardConfiguration.ENABLE_LOGGING)
			Log.d(TAG, msg)
    }
}
